docs/tutorials/how-to-getting-started-with-estimator.ipynb

```diff
@@ -230,7 +218,7 @@
    },
    "source": [
     "<!-- vale IBMQuantum.Spelling = NO -->\n",
-    "The next step is to create an instance of an `Estimator` class. This can be any `Estimator` class that complies with the `Estimator` primitive specification. For simplicity, we will use Qiskit Terra's `qiskit.primitives.Estimator` class, based on the [Statevector construct](https://qiskit.org/documentation/stubs/qiskit.quantum_info.Statevector.html?highlight=statevector#qiskit.quantum_info.Statevector) (algebraic simulation).\n",
+    "The next step is to create an instance of an `Estimator` class, which can be any of the subclasses that complies the base specification. For simplicity, we will use Qiskit Terra's `qiskit.primitives.Estimator` class, based on the [Statevector construct](https://qiskit.org/documentation/stubs/qiskit.quantum_info.Statevector.html?highlight=statevector#qiskit.quantum_info.Statevector) (algebraic simulation).\n",
```

Comment:
"complies" is missing something.  Maybe one of these choices: "comply with" "adhere to" "follow" "work with" 
```